.circleci/config.yml

```diff
@@ -186,7 +180,9 @@ jobs:
         key: apply-repo-{{ .Environment.CIRCLE_SHA1 }}
         paths:
           - ~/project
-    - *setup_database
+    - run:
```

Comment:
Is there a reason you duplicated the steps, rather than replacing the setup_database commands?

Reply:
I saw the comment on the other PR - please ignore - asked and answered!